stock_rule_reserve_max_quantity/models/stock_move.py

```diff
@@ -1,8 +1,9 @@
-# Copyright 2024 Moduon Team S.L.
+# Copyright 2025 Moduon Team S.L.
```

Comment:
suggestion: if you touch copyright headers, always extend, not replace. Here it would be 2024, 2025.